### DISCUSSION TITLE
Unknown application error occurred Runtime.Unknown

### DISCUSSION THREAD

**Discussion by iniTwakkie** [vercel/community#2087] [Help]
### Summary

Any idea where to start, when getting an error like this?
I am trying to deploy my flask application, with simple get and post requests to a database.

My build process is all good and adding in my vercel.json file for inspection..
`{
  "rewrites": [{ "source": "/(.*)", "destination": "/api/app" }]
}
`

### Additional information

Below code is from my app.py file;;

`import sys
from pathlib import Path

sys.path.append(str(Path(__file__).parent))

from flask import Flask
from flask_sqlalchemy import SQLAlchemy
from config import load_config

app = Flask(__name__)
application = app #for the WSGI service
app.config['SQLALCHEMY_DATABASE_URI'] = load_config()
app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False

db = SQLAlchemy(app)

import views

@app.before_first_request
def create_tables():
    db.create_all()

if __name__ == "__main__": 
    app.run()
`

**Comment by MisterChief53** [comment]
Did you ever figure this out? I am trying to debug this and I have no idea how to go about doing this

**Comment by azihassan** [comment]
I ran into a similar issue with Django. `Runtime.Unknown` with no other logs. It turns out I was deploying Django 3.x and the official example had Django 4.x, so an upgrade fixed the runtime unknown issue. In your case maybe try target Flask version 2.2.x since it's what the official example uses : https://github.com/vercel/examples/blob/main/python/flask/requirements.txt

**Comment by salvinoto** [comment]
I am having this issue as well. I have no idea what can cause this because its so vague. Is there some secret log that I'm not seeing?

**Comment by kamath** [comment]
Also running into this issue

**Reply by hassanpasha5630** [reply]
Did you have any luck solving this issue
